src/nvmesh.js

```diff
@@ -3,8 +3,6 @@ import * as pako from "pako";
 import { v4 as uuidv4 } from "uuid";
 import * as cmaps from "./cmaps";
 import { Log } from "./logger";
-import * as mat from "gl-matrix";
-import { NiivueObject3D } from "./niivue-object3D.js";
```

Comment:
NiivueObject3D is a required dependency: used by loadConnectome to generate cylinder and sphere geomtery.